docs/book/src/component-config-tutorial/testdata/project/README.md

```diff
@@ -5,73 +5,70 @@
 // TODO(user): An in-depth paragraph about your project and overview of use
 
 ## Getting Started
-You’ll need a Kubernetes cluster to run against. You can use [KIND](https://sigs.k8s.io/kind) to get a local cluster for testing, or run against a remote cluster.
-**Note:** Your controller will automatically use the current context in your kubeconfig file (i.e. whatever cluster `kubectl cluster-info` shows).
 
-### Running on the cluster
-1. Install Instances of Custom Resources:
+### Prerequisites
+- go version v1.20.0+
+- docker version 17.03+.
+- kubectl version v1.11.3+.
+- Access to a Kubernetes v1.11.3+ cluster.
 
-```sh
-kubectl apply -k config/samples/
-```
-
-2. Build and push your image to the location specified by `IMG`:
+### To Deploy on the cluster
+**Build and push your image to the location specified by `IMG`:**
 
 ```sh
 make docker-build docker-push IMG=<some-registry>/project:tag
 ```
 
-3. Deploy the controller to the cluster with the image specified by `IMG`:
+**NOTE:** This image ought to be published in the personal registry you specified. 
+And it is required to have access to pull the image from the working environment. 
+Make sure you have the proper permission to the registry if the above commands don’t work.
+
+**Install the CRDs into the cluster:**
 
 ```sh
-make deploy IMG=<some-registry>/project:tag
+make install
 ```
 
-### Uninstall CRDs
-To delete the CRDs from the cluster:
+**Deploy the Manager to the cluster with the image specified by `IMG`:**
 
 ```sh
-make uninstall
+make deploy IMG=<some-registry>/project:tag
 ```
 
-### Undeploy controller
-UnDeploy the controller from the cluster:
+> **NOTE**: If you encounter RBAC errors, you may need to grant yourself cluster-admin 
+privileges or be logged in as admin.
+
+**Create instances of your solution**
+You can apply the samples (examples) from the config/sample:
```

Comment:
```suggestion
You can apply the samples (examples) from the config/samples directory:
```

Reply:
I am OK with this one. But you lgtm so, wdyt about doing that in a follow-up?
Could you raise a follow-up with this change?